templates/getting_started_node/.actor/actor.json

```diff
@@ -20,23 +20,20 @@
                     },
                     "display": {
                         "component": "table",
-                        "columns": [
-                            {
+                        "properties": {
+                            "sum": {
                                 "label": "Sum",
-                                "format": "number",
-                                "field": "sum"
+                                "format": "number"
                             },
-                            {
+                            "firstNumber": {
                                 "label": "First number",
-                                "format": "number",
-                                "field": "firstNumber"
+                                "format": "number"
                             },
-                            {
+                            "secondNumber": {
                                 "label": "Second number",
-                                "format": "number",
-                                "field": "secondNumber"
+                                "format": "number"
                             }
-                        ]
+                        }
```

Comment:
Could we potentially leave this out completely? I'm not sure if I want to, it works as documentation, but I have a feeling we wanted to implement the label renaming automatically without the user having to split and capitalize camelCase.